### PR TITLE
Membership engagement copy A/B test in the US

### PR DESCRIPTION
**Pull request by JustinPinner**
## What does this change?

Adds a multivariate test to explore the effects of different messaging to a portion of our US audience.

Explores the hypothesis that "more persuasive copy will improve US membership conversions from impressions"

We've been cleared to engage with 50% of the US audience.

**By default it will be switched off**, so deployment won't alter production behaviour until it is manually switched on.

Also, this is a **change to text only** in the **US only**, no styling will be altered as a result of enabling the test.
## What is the value of this and can you measure success?

Careful use of copy in the message will help to engage more US readers and promote increased membership sign-ups.
## Does this affect other platforms - Amp, Apps, etc?

No. Web only.
## Screenshots

**(variants)**
## screen shot 2016-10-27 at 16 59 33

**(control / notintest)**
screen shot 2016-10-27 at 17 00 15
## Request for comment

@jranks123 @rtyley @svillafe @Ap0c
cc @NataliaLKB for info

### CODE REVIEW

**Comment by Ap0c** [comment]
@SiAdcock would you mind having a look at this?

File: static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-us-message-copy-experiment.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    mediator
+) {
+    return function () {
+        this.id = 'MembershipEngagementUsMessageCopyExperiment';
+        this.start = '2016-10-27';
+        this.expiry = '2016-11-15';
+        this.author = 'Justin Pinner';
+        this.description = 'Test alternate short messages on engagement banner';
+        this.audience = 0.5;
+        this.audienceOffset = 0;
```
Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-us-message-copy-experiment.js]: Note that with this [`audienceOffset`](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/01-ab-testing.md#writing-a-test), this test will coincide with all users participating in [`membership-engagement-warp-factor-one`](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-warp-factor-one.js) and some users participating in [`membership-engagement-message-copy-experiment`](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-message-copy-experiment.js)

Comment (JustinPinner) [inline static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-us-message-copy-experiment.js]: Shouldn't do. Those are UK audience.

**Comment by SiAdcock** [comment]
Looks good to me, as long as you're cool with that one comment about the clashing audience 👍

File: static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-us-message-copy-experiment.js (hunk 1 above)
Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-us-message-copy-experiment.js]: You're right, nice one!

**Comment by JustinPinner** [comment]
Thanks @SiAdcock

**Review by Ap0c — APPROVED** [review]
Also LGTM!

**Comment by JustinPinner** [comment]
Thanks @Ap0c!

**Pull request merged**